### DISCUSSION TITLE
`filter` and `facets` in a range of value

### DISCUSSION THREAD

**Discussion by curquiza** [meilisearch/product#525] [Feedback & Feature Proposal]
Following some user's feedback
- https://github.com/meilisearch/meilisearch/discussions/2062
- https://github.com/meilisearch/meilisearch/discussions/2661
- https://github.com/meilisearch/meilisearch/discussions/2712

People wants to use `filter` and `facets` for a range of values.

Workaround exists for both
- `filter`: https://github.com/meilisearch/meilisearch/discussions/2062#discussioncomment-1947555
- `facets`: https://github.com/meilisearch/meilisearch/discussions/2661#discussioncomment-3407692

However, since it's asked, I open a discussion in case I missed a usecase where the workarounds do not work, or we finally want to add this sugar syntax.

I also open an issue about the filter workaround in the docs since we can help the users with the workaround easily: https://github.com/meilisearch/documentation/issues/1850

**Comment by geminigeek** [comment]
hi,

currently i am doing this running 6 queries in a Promise.all(  1 main query + 5x with custom filter  facet range i want, i only take total count from these ),  my current implementation  works good in my dev environment .

but somehow a native way of doing things should be there.

thanks for this awesome product

**Reply by gmourier** [reply]
Thanks @geminigeek
